### PR TITLE
Pablo/bulma

### PR DESCRIPTION
**Pull request by Pabloj20**
/edit/invoices template. Attached an image of the result
edit_invoices_template

### CODE REVIEW

**Comment by Pabloj20** [comment]
The main files are root.html.heex and edit_invoices.html.heex

File: lib/siwapp_web/templates/layout/root.html.heex [hunk 1]
```diff
@@ -6,26 +6,93 @@
...
+            </a>
+            <a class="navbar-item">
+              Templates
+            </a>
+            <a class="navbar-item">
+              Tags
+            </a>                  
+            <hr class="navbar-divider">
+            <a class="navbar-item">
+              API Token
```
Comment (olirods) [inline lib/siwapp_web/templates/layout/root.html.heex:72]: This is not necessary now

**Review by olirods — APPROVED** [review]

**Review by RodrigoMolinaMadrid — APPROVED** [review]

File: lib/siwapp_web/templates/layout/root.html.heex (hunk 1 above)
Comment (Pabloj20) [reply lib/siwapp_web/templates/layout/root.html.heex:72]: Ok, I will delete those lines

File: lib/siwapp_web/templates/layout/root.html.heex [hunk 2]
```diff
@@ -6,26 +6,93 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
     <%= csrf_meta_tag() %>
     <%= live_title_tag assigns[:page_title] || "Siwapp", suffix: " · Phoenix Framework" %>
-    <link phx-track-static rel="stylesheet" href={Routes.static_path(@conn, "/assets/app.css")}/>
+    <link phx-track-static rel="stylesheet" href={Routes.static_path(@conn, "/assets/app.css")}/>    
     <script defer phx-track-static type="text/javascript" src={Routes.static_path(@conn, "/assets/app.js")}></script>
+    <script src="https://www.jsdelivr.com/package/npm/@creativebulma/bulma-tagsinput/dist/js/bulma-tagsinput.min.js"></script>
```
Comment (peillis) [inline lib/siwapp_web/templates/layout/root.html.heex:11]: remove this

File: mix.exs [hunk 3]
```diff
@@ -50,7 +50,9 @@ defmodule Siwapp.MixProject do
       {:postgrex, ">= 0.0.0"},
       {:swoosh, "~> 1.3"},
       {:telemetry_metrics, "~> 0.6"},
-      {:telemetry_poller, "~> 1.0"}
+      {:telemetry_poller, "~> 1.0"},
+      {:dart_sass, "~> 0.3", runtime: Mix.env() == :dev},
+      {:bulma, "0.9.3"}
```
Comment (peillis) [inline mix.exs:55]: put deps always alphabetically ordered

File: lib/siwapp_web/templates/page/index.html.heex [hunk 4]
```diff
@@ -16,7 +16,7 @@
       <li>
         <a href="https://github.com/phoenixframework/phoenix/blob/v1.6/CHANGELOG.md">v1.6 Changelog</a>
       </li>
-    </ul>
+    </ul>
```
Comment (peillis) [inline lib/siwapp_web/templates/page/index.html.heex:19]: This change shouldn't be

**Review by peillis — CHANGES_REQUESTED** [review]

File: assets/css/phoenix.scss [hunk 5]
```diff
@@ -87,3 +84,7 @@
   text-decoration: none;
   cursor: pointer;
 }
+
+
+
+
```
Comment (peillis) [inline assets/css/phoenix.scss:90]: don't add blank lines

File: lib/siwapp_web/templates/page/invoices.html.heex [hunk 6]
```diff
@@ -0,0 +1,131 @@
...
+        <button class="button is-info">Set Paid</button> 
+        <button class="button is-info">Download PDF</button>               
+        <button class="button is-info">Duplicate</button>					               
+      </div>
+    </div>
+  </div>
+</nav>
+
+<table class="table is-fullwidth is-striped is-bordered is-hoverable">
+	<thead>
```
Comment (peillis) [inline lib/siwapp_web/templates/page/invoices.html.heex:16]: Indent with 4 spaces. This is my last comment about this. You can see this for yourself

File: mix.exs (hunk 3 above)
Comment (peillis) [reply mix.exs:55]: Please do this

File: lib/siwapp/invoices.ex [hunk 7]
```diff
@@ -5,23 +5,26 @@ defmodule Siwapp.Invoices do
 
   import Ecto.Query, warn: false
   alias Siwapp.Repo
-
   alias Siwapp.Schema.Invoices
 
   @doc """
-  Gets a list of invoices by updated date.
+  Gets a list of invoices by updated date
```
Comment (peillis) [inline lib/siwapp/invoices.ex:11]: This file shouldn't be part of your PR, I don't know what you did but this is not your code

File: priv/repo/migrations/.formatter.exs [hunk 8]
```diff
@@ -1,4 +1,4 @@
 [
   import_deps: [:ecto_sql],
-  inputs: ["*.exs"]
+  inputs: ["*.exs", "*.heex"]
```
Comment (peillis) [inline priv/repo/migrations/.formatter.exs:3]: it's useless add this. As far as I know mix format does not change any .heex template

**Review by peillis — CHANGES_REQUESTED** [review]
priv/static/assets directory it's on gitignore, you must not add those files. You probably don't have the right gitignore in the right place

**Pull request closed without merging**